src/data/exchanges/exchanges.yml

```diff
@@ -124,9 +114,9 @@
   name: Trocador
   highlight: false
 -
-  url: "https://huobi.com/"
+  url: "https://htx.com/"
```

Comment:
Maybe they did at some point but I don't think they currently have a DCR spot market. I did find [this page](https://www.htx.com/price/dcr/) though, so I think this should be tagged as instant?

Reply:
They're still reporting volume on CMC and dcrdata through their API, I think it only works for users from ertain countries though.

![image](https://github.com/user-attachments/assets/87619b3e-4f8a-49b1-b4d1-6504d7ed95c3)